packages/smonit/files/etc/smonit/bmx6

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+hook_name="bmx6"
+
+function hook_is_installed() {
+  installed=no
+  [ -x /usr/sbin/bmx6 ] && [ -x /etc/init.d/bmx6 ] && installed=yes
+}
+
+function hook_check() {
+  status=error
+  local bmx6pid=$(cat /var/run/bmx6/pid)
+  [ -n $bmx6pid ] && [ -d /proc/$bmx6pid ] && status=ok
+}
+
+function hook_quickfix() {
+  log="Restarting bmx6 process"
+  PATH="/usr/sbin:/usr/bin:/sbin:/bin"
+  /etc/init.d/bmx6 restart
+}
+
+function hook_longfix() {
```

Comment:
True. It was an example to help understand the concept. But I agree on changing the log message.